backend/tomato/api/elements.py

```diff
@@ -299,6 +299,28 @@ def element_usage(id): #@ReservedAssignment
 	el = _getElement(id)
 	return el.totalUsage.info()	
 
+def element_migrate(id,host): #@ReservedAssignment
```

Comment:
Regular users should not be allowed to migrate elements, so there should be an admin check